### PR TITLE
7243 - fix display name update issue/avatar issue

### PR DESCRIPTION
**Pull request by webdeveloper001**
### Ticket
https://www.notion.so/thegoodparty/

### CODE REVIEW

File: components/profile/ProfileSettingsWrapper/ImageSection.js [hunk 1]
```diff
@@ -84,8 +84,8 @@ function ImageSection({ user, mode = 'desktop', uploadImageCallback }) {
 
           <UserAvatar user={user} size="large" />
         </AvatarWrapper>
-        <H2 style={{ marginLeft: '12px' }}>
-          {fullFirstLastInitials(user.name)}
+        <H2 style={{ marginLeft: '12px', width: '80%', overflow: 'hidden', textOverflow: 'ellipsis' }}>
+          {getName(user)}
```
Comment (tomer-tgp) [inline components/profile/ProfileSettingsWrapper/ImageSection.js:88]: is that the only place we are using it? in the candidate page on the feed we should use that too

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

File: components/profile/ProfileSettingsWrapper/ImageSection.js (hunk 1 above)
Comment (webdeveloper001) [reply components/profile/ProfileSettingsWrapper/ImageSection.js:88]: Got it, Let me check @tomer-tgp

Comment (tomer-tgp) [reply components/profile/ProfileSettingsWrapper/ImageSection.js:88]: I am not seeing that change yet.

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

**Comment by tomer-tgp** [comment]
Also please change getName() with getDisplayName()

File: components/elections/CandidateNewWrapper/RecentlyJoined.js [hunk 2]
```diff
@@ -125,7 +126,7 @@ function RecentlyJoined({
       return supporter.user;
     }
     if (typeof supporter.user === 'object') {
-      return supporter.user.name;
+      return getDisplayName(supporter.user);
```
Comment (tomer-tgp) [inline components/elections/CandidateNewWrapper/RecentlyJoined.js:129]: I think this requires a backend change so we pass the displayName for the supporter as well.

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

File: components/elections/CandidateNewWrapper/RecentlyJoined.js (hunk 2 above)
Comment (tomer-tgp) [reply components/elections/CandidateNewWrapper/RecentlyJoined.js:129]: based on the way we did it on the backend we should go back to what was there before:
`return supporter.user.name;`

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**